docs/src/pages/demos/stepper/stepper.md

```diff
@@ -6,6 +6,64 @@ components: MobileStepper
 
```

Comment:
Need to add the Stepper components above.

Reply:
Fixed in bee14146a